docker-app/qfieldcloud/core/tests/test_jobs.py

```diff
@@ -0,0 +1,173 @@
+import logging
+
+from qfieldcloud.authentication.models import AuthToken
+from qfieldcloud.core.models import (
+    Job,
+    PackageJob,
+    Person,
+    ProcessProjectfileJob,
+    Project,
+)
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.test import APITransactionTestCase
+
+from .utils import (
+    set_subscription,
+    setup_subscription_plans,
+    testdata_path,
+    wait_for_project_ok_status,
+)
+
+logging.disable(logging.CRITICAL)
+
+VALID_LOCALIZED_LAYER_KEY = "apiary_853c29cb_ead8_4932_a60a_5cba33140b3d"
+INVALID_LAYER_KEY = "apiary_e7d1b542_1b1c_422d_ac75_c14d2b54e472"
+INVALID_LOCALIZED_LAYER_KEY = "area_a7b70aff_23f2_4876_9d4e_fa63036df0b2"
```

Comment:
You can change the IDs in the .qgs file to have human readable name, instead of the UUID. If you do so, then the constants are not required anymore.

from `apiary_853c29cb_ead8_4932_a60a_5cba33140b3d` to `valid_localized_point_layer_id`.